src/Subscription.hpp

```diff
@@ -0,0 +1,77 @@
+/**
+ * This software is distributed under the terms of the MIT License.
+ * Copyright (c) 2020 LXRobotics.
+ * Author: Alexander Entinger <alexander.entinger@lxrobotics.com>
+ * Contributors: https://github.com/107-systems/107-Arduino-Cyphal/graphs/contributors.
+ */
+
+#ifndef INC_107_ARDUINO_CYPHAL_SUBSCRIPTION_HPP
+#define INC_107_ARDUINO_CYPHAL_SUBSCRIPTION_HPP
+
+/**************************************************************************************
+ * INCLUDE
+ **************************************************************************************/
+
+#include <memory>
+
+/**************************************************************************************
+ * NAMESPACE
+ **************************************************************************************/
+
+namespace impl
+{
+
+/**************************************************************************************
+ * CLASS DECLARATION
+ **************************************************************************************/
+
+class SubscriptionBase
+{
+public:
+  virtual ~SubscriptionBase() { }
```

Comment:
violation of the rule of 5

Reply:
I don't think this matters for an abstract base class with zero member variables/functions, don't you agree? It's basically an interface and a neat work-around for needing a `std::map<CanardPortId, std::any<Subscription<T>>` within `Node`.

Reply:
It matters always because without disabling the copy constructor/assignment operator you may accidentally copy or move the base and end up with memory corruption. Once you find yourself violating or ignoring the rule of 5, it is a good sign that you are up to something fishy and the code needs refactoring. This is not you case here, just add the missing operators/constructors and you should be good.

Reply:
Done in c653fec4e54675331b88d0092f75c4787470b969.